src/lib.rs

```diff
@@ -807,7 +808,28 @@ mod test {
                 ),
                 // Activity is getting resolved right away as it has been timed out.
                 gen_assert_and_reply(
-                    &job_assert!(wf_activation_job::Variant::ResolveActivity(_)),
+                    &|res| {
+                        assert_matches!(
+                                res.jobs.as_slice(),
+                                [
+                                    WfActivationJob {
+                                        variant: Some(wf_activation_job::Variant::ResolveActivity(
+                                            ResolveActivity {
+                                                activity_id: aid,
+                                                result: Some(ActivityResult {
+                                                    status: Some(activity_result::Status::Failed(ar::Failure {
+                                                        failure: Some(failure)
+                                                    })),
+                                                })
+                                            }
+                                        )),
+                                    }
+                                ] => {
+                                    assert_eq!(failure.message, "Activity task timedOut".to_string());
```

Comment:
"timedOut" instead of "timed out" is a bit odd

Reply:
We could also potentially just add a `TimedOut` status, since we control this now, and that is a lot more clear. I vote we go for that.

Reply:
I agree, copied from java, I think it might be a good time to change it.

Reply:
Addressed